README.md

```diff
@@ -74,6 +81,11 @@ the OnCall tools, use `--disable-oncall`.
 | `get_current_oncall_users`        | OnCall      | Get users currently on-call for a specific schedule                |
 | `list_oncall_teams`               | OnCall      | List teams from Grafana OnCall                                     |
 | `list_oncall_users`               | OnCall      | List users from Grafana OnCall                                     |
+| `get_investigation`               | Sift        | Retrieve an existing Sift investigation by its UUID                     |
+| `get_analysis`                    | Sift        | Retrieve a specific analysis from a Sift investigation                 |
+| `list_investigations`             | Sift        | Retrieve a list of Sift investigations with an optional limit           |
+| `run_error_pattern_logs`          | Sift        | Create a Sift investigation with ErrorPatternLogs check and get results |
+| `run_slow_requests_check`         | Sift        | Create a Sift investigation with SlowRequests check and get results    |
```

Comment:
I think this should be `find_error_pattern_logs` / `find_slow_requests`.

We should change the description to remove Sift there too.